soccer/src/ui/MainWindow.cpp

```diff
@@ -511,21 +538,24 @@ void MainWindow::updateViews() {
     /**************************************************************************/
     /******************** Update referee information **************************/
     /**************************************************************************/
-    _ui.refStage->setText(
-        RefereeModuleEnums::stringFromStage(game_state.raw_stage).c_str());
-    _ui.refCommand->setText(
-        RefereeModuleEnums::stringFromCommand(game_state.raw_command).c_str());
+    // TODO(Kyle): Get these values from the protobuf.
+    _ui.refStage->setText("");
+    _ui.refCommand->setText("");
+    //    _ui.refStage->setText(
+    //        RefereeModuleEnums::stringFromStage(game_state.raw_stage).c_str());
+    //    _ui.refCommand->setText(
+    //        RefereeModuleEnums::stringFromCommand(game_state.raw_command).c_str());
```

Comment:
Commented out code?

Reply:
Ah, yeah. I wanted to bring this up, I'm not sure it even makes sense to display these particular variables - I would prefer to do period/state/restart, I think that's more useful to display.

Reply:
I guess add a TODO to change it then